include/logging/log_output.h

```diff
@@ -50,6 +50,12 @@ extern "C" {
  */
 #define LOG_OUTPUT_FLAG_FORMAT_SYST		BIT(7)
 
+#define LOG_OUTPUT_TEXT 0
+
+#define LOG_OUTPUT_SYST 1
+
+#define LOG_OUTPUT_DICT 2
+
```

Comment:
Recommend adding a comment to indicate where/how these are used.

For example: /* Supported backend logging types for use with log_format_set() */

Reply:
Addressed.